### ISSUE TITLE
pinch zoom doesn't work in ScrollViewer and FlipView on Android 

### ISSUE THREAD

**Issue by TCPcloudDEV** [unoplatform/uno#15010]
### Current behavior

pinch zoom doesn't work on Android when zoom on image itself, but zooming outside of the image on the background works fine

### Expected behavior

expected zoom to work the same ways as it does on Windows 

### Works on UWP/WinUI

Yes

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

Version 17.8.3

**Comment by TCPcloudDEV** [comment]
please try attached example to reproduce it:  

[UnoApp2.zip](https://github.com/unoplatform/uno/files/13877565/UnoApp2.zip)

**Comment by jeromelaban** [comment]
Thanks for the report. Is it only happening when inside the FlipView?

**Comment by TCPcloudDEV** [comment]
no, in my example second page contains ScrollViewer only and I see the same issue

**Comment by MartinZikmund** [comment]
Only related to native
